packages/did-manager/src/id-manager.ts

```diff
@@ -223,7 +223,11 @@ export class DIDManager implements IAgentPlugin {
   ): Promise<any> {
     const identifier = await this.store.getDID({ did })
     const provider = this.getProvider(identifier.provider)
-    const result = await provider.addKey({ identifier, key, options }, context)
+    let result = true
+    // @ts-ignore
```

Comment:
I think it would be best to define options in the argument types `IDIDManagerAddKeyArgs` and so on instead of ts-ignore.

The IDE should show you the typescript warnings. Otherwise, run `pnpm build` locally to see that everything builds.

The PR template has instructions for this in one of the check boxes